### ISSUE TITLE
GO-23 Automaticky zobraziť prevzatú správu

### ISSUE THREAD

**Issue by luciajanikova** [slovensko-digital/govbox-pro#475]
Akutuálne treba buď refresh vlákna alebo prejsť na zoznam správ aby sa prevzatá správa zobrazila po autorizácii doručenky (správa do vlastných rúk). Automaticky sa to nerefreshne/nezobrazí. 
Chceme to ale zmeniť, implementácia by mala byť cez [Action Cable](https://guides.rubyonrails.org/action_cable_overview.html). Po každom ProcessMessageJobe by išiel event na refresh (broadcast page refresh na konkrétny thread).

**Comment by luciajanikova** [comment]
@stage-rl ak by si mal priestor na nejaku ulohu, skusil by si sa na toto pozriet prosim?

**Comment by stage-rl** [comment]
No, pisete sice explicitne Action Cable, pre istotu ale este otazka @luciajanikova @jsuchal , ked uz je Rails 8 predo dvermi, ci neist rovno do morphingu a cieloveho Rails 8 riesenia. Dikes

**Comment by jsuchal** [comment]
morphing prave tuto zmysel uplne nedava, lebo ty nechces vidiet ako sa ti to meni pod rukami, ale byt upozorneny, ze nieco sa tam udialo. 

protipriklad: otvorim si vlakno, dam prevziat spravu. toto bezi async. Ak niekde na koniec vlakna pridas nejaku spravu, tak si to vlastne nevsimnes lebo to je "below fold". Ovela vacsi zmysel dava nejaky notify podobne ako trebars twitter ("new messages") alebo nieco take.

**Comment by stage-rl** [comment]
OK, blbo som sa opytal, co som myslel, je Turbo 8.

T.j. kombinacia https://rubydoc.info/github/hotwired/turbo-rails/Turbo/Broadcastable a https://rubydoc.info/github/hotwired/turbo-rails/main/Turbo%2FStreamsHelper:turbo_stream_from

**Comment by luciajanikova** [comment]
@stage-rl pridala som IP do zoznamu povolenych kvoli stahovaniu sprav + som do DEV schranky (ktora je v seede) naposielala nejake spravy na prevzatie.
